simularium/binding-sim-edu
# Feature/plot formatting

Problem
=======
Estimated review size: small 
The default hover info on the plots wasn't helping people understand them 

closes #140 
closes #141 

Solution
========
This is pretty minor when it comes to functional changes. Mainly is formatting the look of the plots. 

1. added hovertemplates to all plots
2. added another horizontal line because 50% bound is the really important value 
3. added hint text to match the design 
<img width="270" alt="Screenshot 2024-10-11 at 1 53 17 PM" src="https://github.com/user-attachments/assets/786b32bb-e938-460f-9a48-69e60eebf3df">


## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. bun dev
2. follow instructions till you get to press play
3. the plot in the lower right should have hint text
4. the hover of the other ones should be more informative 
5. when you record equilibrium the hint text should go away 

Screenshots (optional):
-----------------------
<img width="286" alt="Screenshot 2024-10-11 at 1 51 34 PM" src="https://github.com/user-attachments/assets/ed0f1384-408f-4779-9c87-dd91951888e4">
<img width="303" alt="Screenshot 2024-10-11 at 1 55 17 PM" src="https://github.com/user-attachments/assets/90666ee1-d82d-4093-9301-73932c113296">
